<launch>

    <!-- HARDWARE -->
        <include file="$(find hardware)/hardware.launch" />
    
    <!-- LOCALIZATION -->
        <arg name="use_scan_matcher" default="true" />
        <arg name="use_slam" default="false" />
    
        <include file="$(find localization)/csm.launch" if="$(arg use_scan_matcher)" />
        <include file="$(find localization)/slam.launch" if="$(arg use_slam)" />
        
    <node name="friction_coeff" pkg="sysid" type="friction_coeff" required="true" output="screen">
        <param name="pose_topic" value="laser_scan_matcher/pose" if="$(arg use_scan_matcher)" />
        <param name="pose_topic" value="hector_mapping/slam_out_pose" if="$(arg use_slam)" />
    </node>

</launch>
